blackbox/handlers/notifiers/telegram.py

```diff
@@ -0,0 +1,44 @@
+import telebot
+from telebot.apihelper import ApiTelegramException
+
+from blackbox.handlers.notifiers._base import BlackboxNotifier
+from blackbox.utils.logger import log
+
+
+STRING_LIMIT = 2000  # Limit output
+
+
+class Telegram(BlackboxNotifier):
+    """ Telegram notifier for Blackbox """
+    required_fields = ("token", "chat_id",)
+
+    def _parse_report(self) -> str:
+        """ Telegram needs a string. Parsing report """
+        data_report = "Blackbox Backup Status:\n"
+
+        for db in self.report.databases:
+            data_report += f"{db.database_id}: \n"
+            if db.success:
+                # Check storages one by one
+                data_report += "".join(
+                    [
+                        f"\U00002705 {storage.storage_id}\n"
```

Comment:
Maybe an inline comment here to explain what these unicode points are? Better yet, you could make variables (constants, maybe?) with descriptive names further up, assign the unicode to those variables, put an inline comment on those with the actual emoji in them and interpolate them. This would be much more readable that way.

```py
CHECKMARK_EMOJI = "\U000023189751"  # ✔️
```